src/controllers/keystore/keystore.ts

```diff
@@ -76,6 +86,8 @@ export class KeystoreController extends EventEmitter {
 
   #storage: Storage
 
+  #keystoreDefaultSeed: string | null = null
+
```

Comment:
Imo we should store the seed in an array in the same way we do for `#keystoreKeys` to be future-proof. Then in `addSeed` we will write:
```
if (#keystoreSeeds.length !== 0)... error state
```